src/ResourceProvider.php

```diff
@@ -38,24 +51,16 @@ public function createImageResource()
     /**
      * @return bool
      */
-    public function imaneeIsSupported()
-    {
-        return ($this->imagickIsLoaded() or $this->gdIsLoaded());
-    }
-
-    /**
-     * @return bool
-     */
-    public function imagickIsLoaded()
+    private function imagickIsLoaded()
```

Comment:
@erikaheidi This and the next method seemed to be used only within this class, so I've made them private